<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>850</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>qudi: Scanner (cwave)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDockWidget" name="dockWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>781</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>630</width>
      <height>711</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="floating">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="dockWidgetContents">
     <widget class="PlotWidget" name="scan_ViewWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>511</width>
        <height>201</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="PlotWidget" name="scan_matrix_ViewWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>511</width>
        <height>221</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>120</y>
        <width>108</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="4" column="0" colspan="2">
        <widget class="QSpinBox" name="scan_cb_min_InputWidget">
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QRadioButton" name="scan_cb_manual_RadioButton">
         <property name="text">
          <string>Man&amp;ual</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QRadioButton" name="scan_cb_centiles_RadioButton">
         <property name="text">
          <string>&amp;Centiles</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="PlotWidget" name="scan_cb_ViewWidget">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="percentLabel2">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;% :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="scan_cb_high_centile_InputWidget">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="scan_cb_low_centile_InputWidget">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="percentLabel1">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;% :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QSpinBox" name="scan_cb_max_InputWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>100000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="PlotWidget" name="wavelength_ViewWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>560</y>
        <width>511</width>
        <height>191</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh_pushButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>720</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>175</width>
        <height>99</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="sweep_checkBox">
           <property name="text">
            <string>sweep</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="regulate_checkBox">
           <property name="text">
            <string>lock at current</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="constLabel_2">
             <property name="text">
              <string>center lambda:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ScienDSpinBox" name="sweepCenterDoubleSpinBox">
             <property name="suffix">
              <string>nm</string>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>515.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
             <property name="value">
              <double>619.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="speedLabel">
             <property name="text">
              <string>sweeps/min :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sweepSpeedSpinBox">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="constLabel">
             <property name="text">
              <string>amplitude:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ScienDSpinBox" name="sweepAmplitudeDoubleSpinBox">
             <property name="suffix">
              <string>nm</string>
             </property>
             <property name="decimals">
              <number>5</number>
             </property>
             <property name="minimum">
              <double>0.000000000000000</double>
             </property>
             <property name="maximum">
              <double>0.010000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.000100000000000</double>
             </property>
             <property name="value">
              <double>0.001000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>12</y>
        <width>66</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>λ, nm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_wavelength">
           <property name="text">
            <string>0 </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>O&amp;ptions</string>
    </property>
    <addaction name="action_Settings"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>660</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_Save"/>
  </widget>
  <action name="action_Settings">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/YY3/artwork/icons/oxygen/22x22/configure.png</normaloff>C:/Users/YY3/artwork/icons/oxygen/22x22/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/YY3/artwork/icons/oxygen/22x22/document-save.png</normaloff>C:/Users/YY3/artwork/icons/oxygen/22x22/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save odmr data</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/YY3/artwork/icons/oxygen/22x22/application-exit.png</normaloff>C:/Users/YY3/artwork/icons/oxygen/22x22/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ScienDSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qtwidgets.scientific_spinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
